index.js

```diff
@@ -100,9 +87,15 @@ module.exports = function enforceHTTPS(options) {
       return next();
     }
 
-    // Check if method should be Forbidden
-    if (!isAllowed(ctx.method, options)) {
-      ctx.response.status = 403;
+    // Check if method should be disallowed (and handle OPTIONS method)
+    if (!redirectStatus[ctx.method]) {
+      if (ctx.method === 'OPTIONS') {
+        ctx.response.status = 200;
```

Comment:
I'm not sure if this is good idea. I think CORS policy should be handled by other middleware otherwise we will need much more settings for this. Making this implicit decision doesn't seems right to me. Is there any other why are you doing this? Can you send some reference to convince me that this is actually expected behavior?